src/OmniSharp.Abstractions/Models/v1/FileMemberElement.cs

```diff
@@ -10,6 +10,8 @@ public class FileMemberElement : IComparable<FileMemberElement>
         public QuickFix Location { get; set; }
 
         public string Kind { get; set; }
+        
+        public ICollection<string> Features { get; } = new List<string>();
```

Comment:
I get the benefit of a structure here. But the interface now is more extensible. Xunit finder can put whatever data in the collection so as other implementation of a `Feature Finder`. The problem with an enum for the type actually makes it hard to expand. For example, the Enum has to be defined in `OmniSharp.Abstractions`. Then every time a new feature is introduced the `OmhniSharp.Abstractions` will have to be updated.